pkg/tests/apis/openapi_test.go

```diff
@@ -74,6 +74,9 @@ func TestIntegrationOpenAPIs(t *testing.T) {
 	}, {
 		Group:   "investigations.grafana.app",
 		Version: "v0alpha1",
+	}, {
+		Group:   "folder.grafana.app",
+		Version: "v0alpha1",
```

Comment:
@Clarity-89 just realised this is already included in the groups array further up. we should remove this duplicate. (maybe also ensure these are unique as part of the test? but that would be extra credit and not necessary 😄)

Reply:
Good catch!